src/commcare_cloud/fab/fabfile.py

```diff
@@ -383,33 +385,47 @@ def prepare_offline_deploy():
 
 
 @task
-def setup_release(keep_days=0):
+def setup_limited_release(keep_days=1):
+    setup_release(keep_days, full_cluster='False')
```

Comment:
nit: I bristle a little at passing string values to be parsed as booleans between functions internally, versus having them accept booleans in the first place. I realize that the outermost later (the @task) has to accept string arguments, but I'd prefer to see those explicitly take a strings, and convert them immediately and call the actual doer function which doesn't concern itself with parsing string arguments. So here I guess that would look like

```python
def parse_int_or_exit(int_string, default):
     if int_string is None:
         return default
     try:
         return int(int_string)
     except ValueError:
         print(red("Unable to parse '{}' into an integer".format(keep_days)))
         exit()


def setup_limited_release(keep_days=None):
    _setup_release(
        keep_days=parse_int_or_exit(keep_days, default=1),  # I think you actually wanted a default of 1 not 0?
        full_cluster=False
    )


def setup_release(keep_days=None):
    _setup_release(
        keep_days=parse_int_or_exit(keep_days, default=0),
        full_cluster=True
    )


def _setup_release(keep_days=0, full_cluster=True):
    # all the stuff, with no string parsing